test/e2e/utils_test.go

```diff
@@ -399,3 +401,27 @@ func verifyCertificateRenewed(ctx context.Context, secretName, namespace string,
 		return true, nil
 	})
 }
+
+// create randomized string
+func randomStr(size int) string {
+	char := "abcdefghijklmnopqrstuvwxyzABCDEFGHIJKLMNOPQRSTUVWXYZ0123456789"
```

Comment:
@geliu2016 
```suggestion
	char := "abcdefghijklmnopqrstuvwxyz0123456789"
```

Reply:
@xingxingxia both changes: char defining and add strings.ToLower are one time updating, strings.ToLower more safer than char defining, otherwise,  "aaa.Author.qe.com" still meet err compare with "aaa.author.qe.com" even define 'aaa' are small letter.

Reply:
Updated.

Reply:
@geliu2016 , it is better to follow above suggestion `char := "abcdefghijklmnopqrstuvwxyz0123456789"`, it is legal to include digits within dns names.